lib/plugins/place_block.js

```diff
@@ -7,7 +7,15 @@ function inject (bot, { version }) {
   const Item = require('prismarine-item')(version)
 
   async function placeBlockWithOptions (referenceBlock, faceVector, options) {
-    if (!bot.heldItem) throw new Error('must be holding an item to place a block')
+    // 0 is placing with mainhand whilst 1 is off-hand
```

Comment:
no need for the comments

Reply:
Do we usually handle errors like this? In the contributors' guide it's says to return the error insted of throwing it.

Reply:
@Zn10plays it does, but I just copied what was written to handle a scenario where bot.helditem is null, so should I change it?

Reply:
Up to you on this one because, .catch will work with either